up-l1/mqtt.adoc

```diff
@@ -35,8 +35,85 @@ MQTT is an OASIS standard messaging protocol for the Internet of Things (IoT). I
 
 For more information, please refer to https://mqtt.org/
 
-
 == Specifications
 
+=== UAttribute Mapping
+
+MQTT 5 supports custom header fields, and we leverage this to map certain UAttribute values into the MQTT header.
+
+[cols="1,1"]
+|===
+| MQTT Header Property Code | Value (key:value)
```

Comment:
Would we be able to include UMessageType in here as well?

Reply:
Sure I will add that to the spec